app/controllers/passwords_controller.rb

```diff
@@ -9,6 +9,20 @@ def edit
     end
   end
 
+  def update
+    @user = find_user_for_update
+
+    if @user.update_password password_reset_params
+      @user.reset_api_key! if params[:password_reset] && params[:password_reset][:reset_api_key].in?([true, '1', 1])
```

Comment:
Can we please use strong param here?

Reply:
Yep fixed in https://github.com/spk/rubygems.org/commit/b529b055f583973024f71459e6ef350a21fea1c1